build.gradle.kts

```diff
@@ -0,0 +1,25 @@
+
+plugins {
+    kotlin("jvm") version "2.0.21"
+}
+
+group = "org.htmlbuilder"
+version = "1.0-SNAPSHOT"
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.9.0")
```

Comment:
Suggestion: Why don't you try gradle version catalogs ? 
https://docs.gradle.org/current/userguide/version_catalogs.html